src/JsonRpc/MessageFactory.php

```diff
@@ -106,7 +111,7 @@ public function create(string $input): iterable
     /**
      * @return class-string<HasMethodInterface>
      */
-    private function getType(string $method): string
+    private function getType(string $method/**/): string
```

Comment:
```suggestion
    private function getType(string $method): string
```

Reply:
Fixed in https://github.com/modelcontextprotocol/php-sdk/pull/42/commits/759a379ec02f9ee475bcc23ae6d939552399137c